ETC Course: 2. The Cypherpunks (#1037)
ethereumclassic/ethereumclassic.github.io: content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md

2. **Hard Money:** To liberate trade from the controls and debasement of government currency, they would study money and try to create a native medium of exchange and store of value for the internet. 
3. **Smart contracts:** To replace the role of prone-to-capture traditional agents and intermediaries, they needed to invent autonomous software entities that could fulfil their functions. 
4. **Trust minimization:** To accomplish decentralization and a peer-to-peer internet with free speech and trade, the overarching principle would be trust minimization, or the significant reduction for the need of third parties to mediate in all human action.

## Cypherpunk People and Inventions: Tim May

![Tim May](./etc-course-tim-may.png)

Tim May was a founding father of the Cypherpunk movement and dedicated his time to explain the philosophy, explain terms and concepts, organize the mailing groups, and even the physical meetings they held in Silicon Valley.

Mr. May was like a role model who provided counsel, motivation, and direction to the group.

> Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the taxman.

Reviewer: ```suggestion
Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the tax man.
```

He passed away in December of 2018.

## Cypherpunk People and Inventions: Adam Back

![Adam Back](./etc-course-adam-back.png)

Adam Back is another Cypherpunk who in 1997 was working on a way to avoid spam in a peer-to-peer email system. If the internet were to be decentralized, then tech company email services should be avoided and each user should be able to prevent spam.

> His solution was a system called HashCash that used proof of work; the heart of the technology in Bitcoin, and the heart of the consensus mechanism that ETC uses today; as a way of blocking spammers.

Reviewer: ```suggestion
As a way of blocking spammers, he designed a system using proof of work called HashCash. Proof of work is at the heart of the technology used in Bitcoin and is the consensus mechanism that ETC uses today.
```
Edited for clarity, to remove awkward punctuation and repeated wording.

## Cypherpunk People and Inventions: Wei Dai

![Wei Dai](./etc-course-wei-dai.png)

Wei Dai discussed the ideas of proof of work and Bit Gold with Nick Szabo in early 1998 and in late 1998 came up with the idea of B-money, a variation of Bit Gold.

The way B-money works as a network protocol is very much how Bitcoin was designed and it is also the peer-to-peer and monetary reward method that Ethereum Classic uses. 

## Cypherpunk People and Inventions: Hal Finney

![Hal Finney](./etc-course-hal-finney.png)

Hal Finney was a legendary Cypherpunk and Bitcoiner as he had also tried to create a proof of work monetary system based on the Bit Gold model that he called RPOW which stands for Reusable Proofs of Work.

Mr. Finney was the first user of Bitcoin and wrote a lot about the network and its future success. 

Like HashCash, Bit Gold, and B-money, some of RPOW’s design was incorporated into Bitcoin by Satoshi Nakamoto and also forms part of ETC’s blueprint.

Hal Finney passed away in August of 2014.

## Next Class

![The four technologies that form part of BTC and ETC.](./etc-course-pioneer-tech-btc-etc.png)

> In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way and formed part of Bitcoin and what is now Ethereum Classic:

Reviewer: ```suggestion
In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way to and formed part of Bitcoin and what is now Ethereum Classic:
```
